kubernetes-maven-plugin/plugin/src/main/java/org/eclipse/jkube/maven/plugin/mojo/develop/UndeployMojo.java

```diff
@@ -83,4 +86,8 @@ protected List<File> getManifestsToUndeploy() {
     return ret;
   }
 
+  @Override
+  protected boolean canExecute() {
+    return super.canExecute() || skipUndeploy;
```

Comment:
We don't know when we'll address https://github.com/eclipse/jkube/issues/1510 ; let's stick to this 

```suggestion
    return  super.canExecute() && !skipUndeploy;
```